app/views/users/index.html.erb

```diff
@@ -1,10 +1,10 @@
-<h1>User Details</h1>
+<h1>List of users</h1>
 <% @users.each do |user| %>
-    <h1>List of Posts for User: <%= user.name %></h1>
-    <ul>
-      @users = User.all
-    </ul>
+  <div><img src="<%= user.photo %>" alt="User Photo"></div>
+    <h1><%= user.name %></h1>
+    <p>Number of post:<%= user.posts_counter %></p>  
   <% end %>
+<h1>Henry</h1>
 <p>Title: Example Post</p>
 <p>Author: John Doe</p>
 <p>Content: Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
```

Comment:
[REQUIRED] Kindly include a link to the user's show page. So that when a user is clicked it redirects to the show page

Reply:
resolved